backend/src/main/java/com/now/naaga/game/application/HintService.java

```diff
@@ -0,0 +1,40 @@
+package com.now.naaga.game.application;
+
+import com.now.naaga.game.application.dto.CreateHintCommand;
+import com.now.naaga.game.application.dto.FindGameByIdCommand;
+import com.now.naaga.game.domain.Direction;
+import com.now.naaga.game.domain.Game;
+import com.now.naaga.game.domain.Hint;
+import com.now.naaga.game.exception.GameException;
+import com.now.naaga.game.exception.GameExceptionType;
+import com.now.naaga.game.repository.HintRepository;
+import com.now.naaga.place.domain.Place;
+import com.now.naaga.place.domain.Position;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional
+@Service
+public class HintService {
+
+    private final HintRepository hintRepository;
+    private final GameService gameService;
+
+    public HintService(final HintRepository hintRepository,
+                       final GameService gameService) {
+        this.hintRepository = hintRepository;
+        this.gameService = gameService;
+    }
+
+    public Hint createHint(final CreateHintCommand command) {
+        final Game game = gameService.findGameById(new FindGameByIdCommand(command.gameId(), command.playerId()));
+        if (!game.canUseMoreHint()) {
+            throw new GameException(GameExceptionType.HINTS_EXHAUSTED);
+        }
+        final Position coordinate = command.coordinate();
+        final Place place = game.getPlace();
+        final Direction direction = Direction.calculate(coordinate, place.getPosition());
+        final Hint hint = new Hint(coordinate, direction, game);
```

Comment:
이 생성 관련 로직은 도메인의 하나의 메서드로 묶여도 되지 않을까요?
```suggestion
        Position position = command.coordinate();
        Hint hint = Hint.of(game, position);
```